content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -6,69 +6,86 @@ aliases:
+In addition to viewing and searching events in the Explorer, you can now graph as a timeseries, top list or table and group the number of events for a given query. See [Event Analytics][2] to learn more.
 
-## Post migration
+You may also generate metrics with 15-month retention from any event search query to create and monitor historical events and alerts.
 
-### Access
+{{< img src="events/generate-metrics.png" alt="Image of metrics with the events search query." >}}
 
-You can access the new Explorer and other features through the Events menu in the main navigation, alongside the existing Stream.
 
-{{< img src="events/guides/navigation.png" alt="Events navigation" style="width:100%;" >}}
+### Graph events in dashboards
 
-### Impact
+{{< img src="events/graph-events.png" alt="Events Analytics" stype="width:40%;" >}}
 
-* Datadog automatically routes Event queries to the correct backend. Old queries continue to go to the old backend, and migrated queries go to the new backend, so you can migrate safely.
-* Event dashboard widgets must use the new query syntax.
+You can now graph events for a given query inside your dashboards as a timeseries graph, query value, top list, table and more.
 
-## Details of the changes
+For example, check out the [Monitor Notifications Overview][3] dashboard, which analyzes monitor alert event trends to help you improve your configuration and reduce alert fatigue.
 
-### Event API
-The [API endpoint to query events][5] does not use any query syntax. It instead uses parameters to include or exclude tags or sources. So, you don't have to change anything in your scripts that use it, unless they refer to status values, which have changed. See [Status remapping](#status-remapping).
+### New event monitors experience
+
+Event monitors have also been ported to use the same standardized set of features as other products (logs, RUM, traces) with new capabilities.
+
+When you create event monitors, the new query search field has autocomplete, rather than the legacy fill-in-the-blank query
+
+{{< img src="events/guides/events-migration-monitor-new.png" alt="New UI for monitor query syntax" style="width:100%;" >}}
 
-### Event query syntax
-The new events query uses the same search query syntax as other products (Logs, RUM), providing a standardized experience across products. The updated search query syntax provides support for more advanced querying through Boolean operators and wildcards. So, the query syntax of your existing event monitors and dashboard widgets must be modified.
+This means you can use complex queries in event monitors with new capabilities such as Boolean operators or wildcards.
 
-#### Status remapping
+## What Changed?
 
+### Event aggregations are no longer performed or displayed in the UI
+Events aggregation will no longer be performed automatically or via the `aggregation_key` attribute. The will also not be displayed in the UI.
```

Comment:
```suggestion
Datadog no longer automatically performs Event aggregation, and no longer groups Events by the `aggregation_key` attribute. The UI no longer displays Event aggregation. 
```
This section is a bit confusing to me. Apologies if I changed the meaning when I reworded it.